_posts/2017-12-18-34C3.md

```diff
@@ -7,10 +7,12 @@ categories: [talk, pinned]
 author: TheAssassin
 ---
 
+<img src="https://engelsystem.de/img/34c3_logo.png" class="postImg">
 In December, the annual [Chaos Communication Congress](https://events.ccc.de/2017/08/09/34c3-presale/) will take place. This is the 34th time hackers from all around the world meet and attend the largest hacker convention in Europe.
 
-Like last year Christian Schabesberger, initial author and primary maintainer of NewPipe and the projects related to it, prepared a talk to inform you about what NewPipe is, how it is developed, and how *you* can participate in the development. This time, we hope that there will be a camera so that a video of the talk can be shot. But be assured, in case there is no such camera, we'll try to record it ourselves.
+Like last year Christian Schabesberger, the initial author and primary maintainer of NewPipe will talk about the deveopment progress of this year, and the roadmap of the next year.
+This time, we hope that there will be a camera so that a video of the talk can be shot.
 
-So, if you happen to attend the 34C3 event, make sure to visit the talk during the first conference day, [December 27, 2017, 2 pm, Saal 3](https://events.ccc.de/congress/2017/wiki/Assembly:Free_Software_Foundation_Europe#Day_1:_Wednesday_27).
+So, if you happen to attend the 34C3 event, please visit the talk during the first conference day, [December 27, 2017, 2 pm, Saal 3](https://events.ccc.de/congress/2017/wiki/index.php/Session:Newpipe).
```

Comment:
I don't like the new wording here, I intended to call on people to attend the talk, and used a slightly more "aggressive" language here. In general, I'd ask you to talk to the author of a post before making changes.

Reply:
@TobiGr feel free to merge this PR, but please make sure you won't change the wording any more, as it's been published for like 5 weeks this way, and I don't like blogs changing the wording of articles like that.

Reply:
Yes, I completely agree with you. Blog posts shouldn't be changed afterwards. In this case it doesn't even make any sense, since the talk already took place. 
I'll merge it anyway, but we should keep this in mind for the future and discuss the posts before publishing them.

Reply:
You are right, but I doubt many people have red the post yet. And I'd like to only add post that have real value to the reader :)
That post about 33c3 happend somwhere in the past so I removed it.

Reply:
@theScrabi it doesn't make sense to *remove* posts, really, just because they're old. How should people know about the old talk, or where to find it?